lucene/core/src/java/org/apache/lucene/index/MergePolicy.java

```diff
@@ -403,6 +409,22 @@ boolean isDone() {
     Optional<Boolean> hasCompletedSuccessfully() {
       return Optional.ofNullable(mergeCompleted.getNow(null));
     }
+
+    void onMergeCommit() {
+    }
+
+    void setMergeReaders(IOContext mergeContext, ReaderPool readerPool) throws IOException {
+      for (final SegmentCommitInfo info : segments) {
+        // Hold onto the "live" reader; we will use this to
+        // commit merged deletes
+        final ReadersAndUpdates rld = readerPool.get(info, true);
```

Comment:
This used to be `getPooledInstance`, but I think this is functionally identical.